src/app/tests/suites/certification/Test_TC_DGGEN_2_1.yaml

```diff
@@ -230,7 +241,7 @@ tests:
               - name: "expectedValue"
                 value: "y"
 
-    - label: "Wait for the commissioned device to be retrieved"
+    - label: "Recommission the DUT after a reboot"
```

Comment:
This is not "recommission".  This is "Establish CASE".

Reply:
Reverted back to the previous version.